Makefile

```diff
@@ -106,11 +106,21 @@ dist/package-$(PACKAGE_VERSION).tar.gz: check test
 	flit build --setup-py --format sdist
 
 # Build the HTML documentation from the package's source.
-.PHONY: docs
-docs: docs/_build/html/index.html
+.PHONY: docs-html
+docs-html: docs/_build/html/index.html
 docs/_build/html/index.html: check test
 	$(MAKE) -C docs/ html
 
+# Build the Markdown documentation from the package's source.
+.PHONY: docs-md
+docs-md: docs/_build/markdown/index.md
+docs/_build/markdown/index.md: check test
+	$(MAKE) -C docs/ markdown
```

Comment:
Currently the Action [renames the `index.md` file to `Home.md`](https://github.com/lecram/python-package-template/blob/81e626494b384285a7b06f5eaa9214e7db698739/.github/workflows/wiki-documentation.yaml#L32-L34):
```yaml
      run: |
        make docs-md
        mv docs/_build/markdown/index.md docs/_build/markdown/Home.md
```
Looking at my own [Bookalope wiki](https://github.com/bookalope/Bookalope/wiki), however, I notice that _all_ pages should be capitalized whereas for the HTML output they’re not. Leave it for later, or should we discuss that filename difference now?

Also, should we consider the Wiki [footer and sidebar](https://docs.github.com/en/communities/documenting-your-project-with-wikis/creating-a-footer-or-sidebar-for-your-wiki#creating-a-footer-or-sidebar-locally), though these two files would probably _not_ be useful for HTML output?

Reply:
The original idea was keeping `make docs-md` just for generic Markdown docs (not necessarily Github Wiki). This is why the "index.md" -> "Home.md" rename was only done in the Action. There are probably other caveats of Github Wiki that we wanna handle (links, sidebar, etc). I haven't tested this extensively yet. Maybe at some point we should add a `make docs-github` command which would depend on `docs-md` and deal with the Github-specific stuff. Probably something for a later PR.

Reply:
Yes I agree. Let’s get the feature working in a simple and stable way in this PR and then we can polish and expand late 👍🏼